config/bidderinfo_test.go

```diff
@@ -928,6 +928,303 @@ func TestBidderInfoValidationNegative(t *testing.T) {
 				errors.New("The endpoint: incorrect for bidderB is not a valid URL"),
 			},
 		},
+		{
```

Comment:
Could we please write one test case that passes?

Reply:
Positive test is already covered in [TestBidderInfoValidationPositive](https://github.com/prebid/prebid-server/blob/master/config/bidderinfo_test.go#L499)